File: cpp/hello-world/hello_world.cpp
#include "hello_world.h"
// exercism, cpp track, exercise 1
// Z Knight, 2019.09.07

// Use everything from the 'std' namespace.
// This lets us write 'string' instead of 'std::string'.
using namespace std;

namespace hello_world {

// Define the function itself. This could have also been written as:
// std::string hello_world::hello()
string hello() {
    // Return the string we need.
    return "Hello, World!";
}

}  // namespace hello_world

File: python/hello-world/hello_world.py
#! /usr/bin/env python
"""
    Name:
        hello_world.py
    Purpose:
        exercism, python track, exercise 1.
        Write a function (named 'hello') that returns the string "Hello, World!"
    Modificaiton History:
        Written by Z Knight, 2019.09.07
"""
def hello():
    return("Hello, World!")
    
###############################################################################
# main

if __name__=="__main__":
    response = hello()
    print("response = {}".format(response))

